core/src/main/scala/io/chrisdavenport/github/endpoints/repositories/Merge.scala

```diff
@@ -0,0 +1,40 @@
+package io.chrisdavenport.github.endpoints.repositories
+
+import cats.data._
+import cats.effect._
+import cats.implicits._
+
+import io.chrisdavenport.github.Auth
+import io.chrisdavenport.github.data.Repositories._
+import io.chrisdavenport.github.internals.GithubMedia._
+import io.chrisdavenport.github.internals.RequestConstructor
+
+import org.http4s._
+import org.http4s.client._
+import org.http4s.implicits._
+
+/**
+ * The Repo Merging API supports merging branches in a repository. This accomplishes essentially
+ * the same thing as merging one branch into another in a local repository and then pushing to GitHub.
+ *
+ * See the Repository Merging API for more details.
+ **/
+object Merge {
+
+  /**
+   * Perform a merge in a repository - https://developer.github.com/v3/repos/merging/#perform-a-merge
+   * POST /repos/:owner/:repo/merges
+   **/
+  def perform[F[_]: Sync](
```

Comment:
```suggestion
  def merge[F[_]: Sync](
```

Seems more intuitive to me and matches other endpoints, but I'm intrigued if you think.

Reply:
It seems redundant to me `io.chrisdavenport.github.repositories.Merge.merge(...)`, but no more reason :smile: